test/native/integration-test-helpers/setup-media-picker.js

```diff
@@ -34,16 +36,25 @@ export const setupMediaPicker = () => {
 				mediaPickerCallback
 			),
 		mediaPickerCallback: async ( ...mediaItems ) =>
-			act( async () =>
+			act( async () => {
+				const items = mediaItems.map(
+					( {
+						localId,
+						localUrl,
+						type = 'image',
+						url,
+						id,
+						metadata,
+					} ) => ( {
+						type,
+						url: url ?? localUrl,
+						id: id ?? localId,
```

Comment:
Note that now if the media item has `url` and `id` properties, they will be used instead of local ones.